From 16a6e0b5bad29fcc2500d6bd82ed6db4a1f520fc Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 20 Jul 2018 18:33:01 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3308: Raise voltage for 1008MHz,
 1200MHz and 1296MHz

In order to cover the chips passed cp test program(1.3g 1.175v).

Change-Id: I4e19aefd914258e8d1e6d331b6f584aa7d0c4822
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index 276247d21db9..3668e86e42fe 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -119,18 +119,18 @@
 		};
 		opp-1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
-			opp-microvolt = <1100000 1100000 1340000>;
+			opp-microvolt = <1150000 1150000 1340000>;
 			clock-latency-ns = <40000>;
 		};
 		opp-1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
-			opp-microvolt = <1200000 1200000 1340000>;
+			opp-microvolt = <1250000 1250000 1340000>;
 			clock-latency-ns = <40000>;
 			status = "disabled";
 		};
 		opp-1296000000 {
 			opp-hz = /bits/ 64 <1296000000>;
-			opp-microvolt = <1250000 1250000 1340000>;
+			opp-microvolt = <1300000 1300000 1340000>;
 			clock-latency-ns = <40000>;
 			status = "disabled";
 		};
-- 
2.35.3

